content/overview/security/fedramp-tracker.md

```diff
@@ -12,30 +12,24 @@ aliases:
 ---
 
 
-cloud.gov has a [Provisional Authority to Operate (P-ATO) at the Moderate impact level from the FedRAMP Joint Authorization Board (JAB)](https://marketplace.fedramp.gov/#/product/18f-cloudgov).
+cloud.gov has a [Provisional Authority to Operate (P-ATO) at the Moderate impact level from the FedRAMP Joint Authorization Board (JAB)](https://marketplace.fedramp.gov/#/product/18f-cloudgov). This means cloud.gov has undergone a significant, thorough security and compliance review so that your agency doesn't have to.
```

Comment:
"Doesn't have to" seems to end a bit abruptly. Maybe "This means cloud.gov has undergone a significant, thorough security and compliance review, so that your agency can use that review instead of conducting its own." ?

Reply:
> so that your agency can focus on reviewing the parts of the system that serve your mission.